CLAUDE.md

```diff
@@ -0,0 +1,33 @@
+# LangGraphJS Development Guide
+
+## Build & Test Commands
+- Build: `yarn build`
+- Lint: `yarn lint` (fix with `yarn lint:fix`)
+- Format: `yarn format` (check with `yarn format:check`)
+- Test: `yarn test` (single test: `yarn test:single /path/to/yourtest.test.ts`)
+- Integration tests: `yarn test:int` (start deps: `yarn test:int:deps`, stop: `yarn test:int:deps:down`)
+
+## Code Style Guidelines
+- **TypeScript**: Target ES2021, NodeNext modules, strict typing enabled
+- **Formatting**: 2-space indentation, 80 char width, double quotes, semicolons required
```

Comment:
probably good if it tries to keep to 80 or 120 char width, but we don't enforce this